src/quickstart/05-understanding-air-quality.mdx

```diff
@@ -0,0 +1,60 @@
+## Understanding Air Quality
+
+Air quality is the result of several factors. The single largest contributing
+factor is particulate matter, or PM. According to the EPA, "PM stands for
+particulate matter (also called particle pollution): the term for a mixture of
+solid particles and liquid droplets found in the air."
+
+The Airnote samples and reports three different levels of particulate matter,
+PM<sub>10</sub>, PM<sub>2.5</sub>, and PM<sub>1.0</sub>.
+
+![Illustration showing the scale of particulate matter](/images/guides/airnote/pm2.5_scale_graphic-color_2.png)
+
+_[Graphic courtesy of EPA.gov](https://www.epa.gov/sites/production/files/2016-09/pm2.5_scale_graphic-color_2.jpg)_
+
+### PM<sub>10</sub>
+
+Inhalable particles, with diameters that are generally 10
+micrometers and smaller. PM<sub>10</sub> is a mixture of materials that can
+include smoke, soot, dust, salt, acids, and metals. Particulate matter also
+forms when gases emitted from motor vehicles and industrial machinery undergo chemical
+reactions in the atmosphere.
+
+### PM<sub>2.5</sub>
+
+Fine inhalable particles, with diameters 2.5 micrometers and
+smaller. Fine particulate matter (PM<sub>2.5</sub>) is an air pollutant that is
+a concern for people's health when levels in air are high. PM<sub>2.5</sub> are
+tiny particles in the air that reduce visibility and cause the air to appear
+hazy when levels are elevated. Outdoor PM<sub>2.5</sub> levels are most likely
+to be elevated on days with little or no wind or air mixing.
+
+### PM<sub>1.0</sub>
+
+Described as extremely fine particulate matter, PM<sub>1.0</sub> is much less
+frequently talked about, as there is little available data. However, it is
+believed to have even more significant health impacts than PM<sub>2.5</sub> and
+PM<sub>10</sub>. However, scientists still aren't clear about the health
+implications of these particles.
+
+### EPA Standards (December 14th, 2012)
+
+|      PM      | Safe Levels (&micro;g/m&sup3;) |
+| :----------: | :----------------------------: |
+| 1.0 (24-hrs) |               --               |
+| 1.0 (Annual) |               --               |
+| 2.5 (24-hrs) |               35               |
+| 2.5 (Annual) |               12               |
+| 10 (24-hrs)  |              150               |
+| 10 (Annual)  |              50\*              |
+
+> _\* Originally set at 50 &micro;g/m&sup3;, the annual PM10 standard was revoked
+> because of a lack of evidence establishing a link between long-term exposure to
+> coarse particles and health problems._
+
+_**NOTE:** The information regarding particulate matter, and its effects, has
+been taken in whole or part from
+[EPA.gov: Particulate Matter (PM) Basics](https://www.epa.gov/pm-pollution/particulate-matter-pm-basics) and
+[kaiterra: A Guide to Understanding Particulate Matter (PM)](https://learn.kaiterra.com/en/air-academy/particulate-matter-pm)._
+
+<hr />
```

Comment:
Trivial, but probably a little cleaner if you did these lines via CSS instead of having them in the bottom of the Markdown content.

Reply:
Actually I’d just put them in `quickstart/_main.mdx`. Also fyi `---` in Markdown makes an `<hr>`.